jetty-alpn/jetty-alpn-openjdk8-client/src/main/java/org/eclipse/jetty/alpn/openjdk8/client/OpenJDK8ClientALPNProcessor.java

```diff
@@ -33,11 +40,28 @@
 {
     private static final Logger LOG = Log.getLogger(OpenJDK8ClientALPNProcessor.class);
 
+    private Method alpnProtocols;
+    private Method alpnProtocol;
+
     @Override
     public void init()
     {
         if (JavaVersion.VERSION.getPlatform() != 8)
             throw new IllegalStateException(this + " not applicable for java " + JavaVersion.VERSION);
+
+        try
+        {
+            // JDK 8u252 has the JDK 9 ALPN API backported.
+            // Use reflection so we can build with a JDK version less than 8u252.
+            alpnProtocols = SSLParameters.class.getMethod("setApplicationProtocols", String[].class);
+            alpnProtocol = SSLEngine.class.getMethod("getApplicationProtocol");
+            return;
+        }
+        catch (NoSuchMethodException ignored)
+        {
```

Comment:
Probably should do a LOG.ignore here so we can see the exception if we really want to

Reply:
Done